src/deployments/cdk/src/common/interface-endpoints.ts

```diff
@@ -67,7 +67,7 @@ export class InterfaceEndpoint extends cdk.Construct {
         },
       ],
       hostedZoneConfig: {
-        comment: `zzEndpoint - ${serviceName}`,
+        comment: `Endpoint - ${serviceName}`,
```

Comment:
- The comment currently starts with a 'zz' as we had customer requests to allow sort by comment forcing all the 'endpoints' to the bottom of the list.
- Thoughts?

Reply:
sure

Reply:
Reverted